model/Service/PauseService.php

```diff
@@ -30,6 +30,8 @@
 
 class PauseService
 {
+    public const PAUSE_REASON_CONCURRENT_TEST = 'PAUSE_REASON_CONCURRENT_TEST';
```

Comment:
Is it still needed?

- [ ] remove if nod needed anymore - No usages in this very extension at least

Reply:
This is used by `ltiDeliveryProvider` to store why a delivery has been paused, and then from the feedback screen (in  the same extension) to known which message to show (just _paused_ vs _paused because a concurrent test_). Additionally, it is used similarly from the `ltiTestReview` extension (to indicate there is a concurrent session, in this case the "review session").

It could be moved to `ltiDeliveryProvider` (since the only other usage is from `ltiTestReview`, and that already declares a dependency on `ltiDeliveryProvider`), but I added this here because I considered the reason for pausing  a test part of the model of the `(Qti)Test` itself.

Reply:
Got it. The thing is that what we pause is a delivery execution, not the `QTI test`. I think the boundaries here are already overlapping a bit.

Also what is there on test review, probably could go to another extension (or this one) https://github.com/oat-sa/extension-lti-test-review/pull/135#discussion_r1406106289

I do not have a strong opinion on this as ltiDeliveryProvider, taoDelivery, taoDeliveryRdf, testQti, ltiTestReview, etc are kind of talking about similar thing but segregated. 

So, I accept suggestions on this one. If it is already in use, we can keep it. And if this is part of the parameters used by `PauseService`, it is ok to be here.